### PR TITLE
removed TODOs in dtc.h

### PR DESCRIPTION
**Pull request by raticate**
added corresponding documentation

### CODE REVIEW

**Comment by fmontoto** [comment]
Do not add a new file, change the existing dtc.h.in.

**Comment by raticate** [comment]
corrected that

**Comment by raticate** [comment]
true.

File: src/include/dtc.h.in [hunk 1]
```diff
@@ -91,12 +91,14 @@ dtc_ctx_t *dtc_init(const char *config_file, int *err);
...
- * @param info //TODO
+ * @param bit_size Desired size of the RSA modulus of the generated keys.
+ * @param threshold threshold parameter of the keys. Must be greater than cant_nodes / 2 and
+ *      lesser or equal to cant_nodes.
+ * @param cant_nodes Number of nodes. Must be greater than one.
+ * @param info Pointer to the key_meta_info structure that will contain the public
+        information of the keys (modulus, public exponent, etc). 
  *
- * @return -1 if key_id is already used. //TODO
+ * @return @return DTC_ERR_NONE on success, a proper error code otherwise. -1 if key_id is already used.
```
Comment (fmontoto) [inline src/include/dtc.h.in]: return is duplicated :(

Comment (raticate) [inline src/include/dtc.h.in]: -_-

**Pull request merged**